### PR TITLE
Bump to mesos 0.27.0

### PR DESCRIPTION
**Pull request by aquamatthias**
Add the mesosphere jessie testing repository, in order to try mesos RC releases.

### CODE REVIEW

File: Dockerfile.development [hunk 1]
```diff
@@ -8,8 +8,9 @@ FROM java:8-jdk
 
 RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E56151BF && \
     echo "deb http://repos.mesosphere.com/debian jessie main" | tee /etc/apt/sources.list.d/mesosphere.list && \
+    echo "deb http://repos.mesosphere.com/debian jessie main" | tee -a /etc/apt/sources.list.d/mesosphere.list && \
```
Comment (gkleiman) [inline Dockerfile.development]: Is this line really needed? The previous line adds exactly the same repo to exactly the same file.

**Comment by gkleiman** [comment]
@aquamatthias the new repository is actually not new. Maybe you forgot to replace `main` with `testing`?

**Comment by aquamatthias** [comment]
@gkleiman You are right - the merge operation went wrong. Thanks.

**Comment by gkleiman** [comment]
@aquamatthias :rocket:

**Pull request merged**